docs/attributes.md

```diff
@@ -0,0 +1,94 @@
+### `#[StateId]`
```

Comment:
Second argument sets the alias on the collection. Third argument determines if Verbs should automatically create an ID if one isn't passed into a nullable property.